Automattic/wp-calypso
# Support product and plan slugs directly in the component without facade

This PR is a response to the feedback received in #38474 

It adds a support for plan and product slugs as they are in `wp-calypso` or `jetpack` so that no longer [a mapping function](https://github.com/Automattic/wp-calypso/pull/38474/files#diff-a33b3012cc6794a297b3c589a2df0b3cR544-R594) is needed in the consumer.

Once this is reviewed, approved and merged, I'll go though #38474 and simplify all the usages of `ProductIcon` component.

#### Changes proposed in this Pull Request

* Support product and plan slugs directly in the component without a facade

#### Testing instructions



* Go to http://calypso.localhost:3000/devdocs/design/product-icon
* Confirm that all icons are properly displayed and there are no errors in the console

Fixes n/a
